x-pack/plugins/observability/public/components/threshold/components/expression_row.tsx

```diff
@@ -144,87 +134,48 @@ export const ExpressionRow: React.FC<ExpressionRowProps> = (props) => {
+    },
+    [debouncedLabelChange, expression]
+  );
+
   return (
     <>
       <EuiFlexGroup gutterSize="xs">
-        <EuiFlexItem grow={false}>
-          <EuiButtonIcon
-            iconType={isExpanded ? 'arrowDown' : 'arrowRight'}
-            onClick={toggle}
-            data-test-subj="thresholdRuleExpandRow"
-            aria-label={i18n.translate(
-              'xpack.observability.threshold.rule.alertFlyout.expandRowLabel',
-              {
-                defaultMessage: 'Expand row.',
-              }
-            )}
-          />
-        </EuiFlexItem>
         <EuiFlexItem grow>
-          <StyledExpressionRow style={{ gap: aggType !== 'custom' ? 24 : 12 }}>
-            <StyledExpression>
-              <EuiExpression
-                data-test-subj="thresholdRuleCustomEquationWhen"
-                description={i18n.translate(
-                  'xpack.observability.thresholdRule.expressionItems.descriptionLabel',
-                  {
-                    defaultMessage: 'when',
-                  }
-                )}
-                value={aggregationType.custom.text}
-                display={'inline'}
-              />
-            </StyledExpression>
-            {!['count', 'custom'].includes(aggType) && (
-              <StyledExpression>
-                <OfExpression
-                  customAggTypesOptions={aggregationType}
-                  aggField={metric}
-                  fields={normalizedFields}
-                  aggType={aggType}
-                  errors={errors}
-                  onChangeSelectedAggField={updateMetric}
-                  helpText={
-                    <FormattedMessage
-                      id="xpack.observability.threshold.rule.alertFlyout.ofExpression.helpTextDetail"
-                      defaultMessage="Can't find a metric? {documentationLink}."
-                      values={{
-                        documentationLink: (
-                          <EuiLink
-                            data-test-subj="thresholdRuleExpressionRowLearnHowToAddMoreDataLink"
-                            href="https://www.elastic.co/guide/en/observability/current/configure-settings.html"
-                            target="BLANK"
-                          >
-                            <FormattedMessage
-                              id="xpack.observability.threshold.rule.alertFlyout.ofExpression.popoverLinkLabel"
-                              defaultMessage="Learn how to add more data"
-                            />
-                          </EuiLink>
-                        ),
-                      }}
-                    />
-                  }
-                  data-test-subj="thresholdRuleOfExpression"
-                />
-              </StyledExpression>
-            )}
-            {criticalThresholdExpression}
-          </StyledExpressionRow>
+          <StyledExpressionRow style={{ gap: aggType !== 'custom' ? 24 : 12 }} />
 
           {aggType === Aggregators.CUSTOM && (
```

Comment:
Is this condition still needed, or can we get rid f it?

Reply:
Deleted